dot/rpc/modules/api.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/ChainSafe/gossamer/lib/genesis"
 	"github.com/ChainSafe/gossamer/lib/grandpa"
 	"github.com/ChainSafe/gossamer/lib/runtime"
+	rtstorage "github.com/ChainSafe/gossamer/lib/runtime/storage"
```

Comment:
nit Why not without import alias `rtstorage`? I don't see any conflicting `storage` import 🤔